view.go

```diff
@@ -204,10 +204,11 @@ func (v *View) Name() string {
 // setRune sets a rune at the given point relative to the view. It applies the
 // specified colors, taking into account if the cell must be highlighted. Also,
 // it checks if the position is valid.
-func (v *View) setRune(x, y int, ch rune, fgColor, bgColor Attribute) error {
+// Returns the set rune (!may be different from input)
```

Comment:
The ! seems a bit wired
```suggestion
// Returns the set rune (may be different from input)
```

Reply:
This change will likely get reverted/removed pending the result of the UTF-8 character PR